### PR TITLE
repro case for #6168

### PR DESCRIPTION
**Pull request by Alizter**

### CODE REVIEW

File: test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+let () =
+  let _ = Sys.command "touch file" in
```
Comment (rgrinberg) [inline test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml:2]: Why is creating this file needed?

Comment (rgrinberg) [reply test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml:2]: Can't the bug be reproduced with a broken symlink?

Comment (Alizter) [reply test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml]: I don't know how to make a broken one.

File: test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+let () =
+  let _ = Sys.mkdir "fakenode_modules" 0o777  in
+  let _ = Sys.mkdir "fakenode_modules/foo" 0o777 in
+  let _ = Unix.symlink "file" "./fakenode_modules/foo/bar" in
```
Comment (rgrinberg) [inline test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml:4]: Why do you use this `let _` form? All these functions return `unit` and should just be terminated with semicolons.

Comment (Alizter) [reply test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml:4]: I had the `Sys.command` before which returned an `int` which is why I did that.

File: test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+let () =
+  Sys.mkdir "fakenode_modules" 0o777;
+  Sys.mkdir "fakenode_modules/foo" 0o777;
+  Unix.symlink "file" "./fakenode_modules/foo/bar";
+  ()
```
Comment (rgrinberg) [inline test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml:5]: why do you have this terminating `()`?

Comment (Alizter) [reply test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml]: It's just a style to help indicate the end of a bunch of semicolons. I've removed it.

**Pull request merged**